README.md

```diff
@@ -63,11 +63,18 @@ See the [dedicated page](guides/Create-a-deploy-user.md).
 
 ## Renew SSL certificates
```

Comment:
> Move `Set up a SSL certificate` section next to this `Renew SSL certificates`?

Reply:
Thanks @sandcha, I'm not the author of that line